### PR TITLE
Use Redux in React components

### PR DESCRIPTION
**Pull request by Mujeeb4582**
## Task completed
- Connect the Redux in React components
- Display the booklist  using redux-store
- Add the new book feature using redux-store
- Add the remove book feature using redux-store
- Add the features to the check-status  button in the categories page using redux-store

### CODE REVIEW

File: src/components/Books/form.jsx [hunk 1]
```diff
@@ -1,16 +1,31 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/books/books';
 
-const Form = () => (
-  <div className="form-container">
-    <h2 className="form-title">ADD NEW BOOK</h2>
+const Form = () => {
```
Comment (FrederickMih) [inline src/components/Books/form.jsx:6]: - [ ] Based on the requirements of this project, this component should contain a local state with a title and author. They should have their input that updates the local state every time they update(onChange). Kindly fix this. You can try out this [article](https://blog.logrocket.com/component-state-local-state-redux-store-and-loaders/)

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @Mujeeb4582 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Descriptive and professional README.md file :+1: 
- useSelector has been used to display books from the redux store :+1: 
- Fantastic commit history :+1: 
- PR is descriptive :+1: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

N/A

_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books/form.jsx (hunk 1 above)
Comment (peterrobert) [reply src/components/Books/form.jsx:6]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Mujeeb4582, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**